nixos/modules/config/console.nix

```
@@ -8,14 +8,22 @@ let
 
   makeColor = i: concatMapStringsSep "," (x: "0x" + substring (2*i) 2 x);
 
-  isUnicode = hasSuffix "UTF-8" (toUpper config.i18n.defaultLocale);
+  isUnicode = '' \
+    LOCALE_ARCHIVE=${config.i18n.glibcLocales}/lib/locale/locale-archive \
+    LANG=${config.i18n.defaultLocale} \
```

Maybe set
```suggestion
    LC_IDENTIFICATION=${config.i18n.defaultLocale} \
```
Depending on https://github.com/NixOS/nixpkgs/pull/183904/files#r935826135